lib/AgrirouterClient/src/Protobuf/meson.build

```diff
@@ -1,7 +1,6 @@
-protoc_exe = find_program('protoc')
-
-pymodule = import('python3')
-python3 = pymodule.find_python()
+import('python')
```

Comment:
This import is no longer used, so it is dead code and can be eliminated. :)

Reply:
Yes, thats dead code. Thank you for the review